examples/global_tables/README.md

```diff
@@ -0,0 +1,61 @@
+[![Build Status](https://github.com/boldlink/terraform-aws-dynamodb/actions/workflows/pre-commit.yml/badge.svg)](https://github.com/boldlink/terraform-aws-dynamodb/actions)
+
+[<img src="https://avatars.githubusercontent.com/u/25388280?s=200&v=4" width="96"/>](https://boldlink.io)
+
+# Example with DynamoDB Global tables configuration
+
+**NOTE**: The check `CKV_AWS_119 #Ensure DynamoDB Tables are encrypted using a KMS Customer Managed CMK` was skipped in this example because we are encrypting both replicas
```

Comment:
Add at the end of the note " for prod it's recornented you use a Cmk and not a AWS owned or AWS CMK key. I